configure

```diff
@@ -2537,6 +2548,17 @@ if test -n "$_host"; then
 			_seq_midi=no
 			_port_mk="backends/platform/dingux/dingux.mk"
 			;;
+		raspberrypi)
+			# We should have setup raspberrypi-sdl-config with modified prefix as part of 
+			# the cross-building enviroment.	
```

Comment:
That statement is rather ambiguous (and contains a typos ;-). Perhaps you meant to say something like this? `For proper cross-compilation, we assume that the cross-building environment provides a raspberrypi-sdl-config executable with modified prefix.` or so?

But even then it is not completely clear to me whether this is appropriate. E.g. is it customary that such an executable und that name is provided? At least in the past, most cross environments I dealt with just called it `sdl-config` (and it seems that on the ScummVM buildbot, this is still the case, see e.g. http://buildbot.scummvm.org/builders/master-gp2xwiz/builds/6166/steps/configure/logs/stdio )

But if that's really as the comment suggestings, then `find_sdlconfig` won't find `raspberrypi-sdl-config`. For that, you'd have to do something similar to this (taken from `configure`):
```
		*mingw32*)
			_sdlconfig=$_host-sdl-config
```